DevAcademy/ukol1.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  ukol1.swift
+//  DevAcademy
+//
+//  Created by To je jedno on 18.07.2023.
+//
+
+import Foundation
+
+enum Kind: String{
+    case divadlo = "Divadlo"
+    case galerie = "Galerie"
+    case hub = "Hub"
+    case hudebniKlub = "Hudebni klub"
+    case kino = "Kino"
+    case knihovna = "Knihovna"
+    case koncertniHala = "Koncertní hala"
+    case kulturniCentrum = "Kulturní centrum"
+    case kulturniPamátka = "Kulturní památka"
+    case letniKino = "Letní kino"
+    case muzeum = "Muzeum"
+    case podnikSLulturnimProgramem = "Podnik s kulturním programem"
+    case vystaviste = "Výstaviště"
+    case ostatni = "Ostatní"
+}
+enum PossibleKind: RawRepresentable {
+    init?(rawValue: String) {
+        guard let kind = Kind(rawValue: rawValue) else{
+            self = .unknown(rawValue)
+            return
+        }
+        self = .kind(kind)
+        
+    }
+    
+    var rawValue: String{ //Read-only computer property
```

Comment:
Jen pro upresneni, mozna jde jenom typo v komentari `Read-only computer property` -> `Read-only computed property`. 

Jedna se o computed nebo-li pocitanou promennou. Jde o to, ze ta promenna si vlastne nedrzi zadnou hodnotu, ale pri kazdem volani je znova pocitana (vzdy kdyz nekde zavolas `.rawValue`, probehne ten obsah zavorky a spocita se hodnota te promenne).